pkg/tsdb/cloudwatch/log_actions.go

```diff
@@ -57,14 +57,10 @@ func (e *cloudWatchExecutor) executeLogActions(ctx context.Context, req *backend
 		eg.Go(func() error {
 			dataframe, err := e.executeLogAction(ectx, logsQuery, query, req.PluginContext)
 			if err != nil {
-				var AWSError *AWSError
-				if errors.As(err, &AWSError) {
-					resultChan <- backend.Responses{
-						query.RefID: backend.DataResponse{Frames: data.Frames{}, Error: AWSError},
-					}
-					return nil
+				resultChan <- backend.Responses{
+					query.RefID: backend.DataResponse{Frames: data.Frames{}, Error: err},
 				}
-				return err
```

Comment:
basically before we returned a raw error unless it was formatted in a certain way, now we always return an response with an error. 

I think this is probably always desirable? It lets us connect the error to the ref id, and ensures we display the error properly. That said it looks like our frontend code at least used to expect raw errors more.